### ISSUE TITLE
If a file is removed during the sync process, livepush will cancel with an error

### ISSUE THREAD

**Issue by CameronDiver** [balena-io-modules/livepush#48]
ref: https://forums.balena.io/t/test-livepush-change/31462/14

```
[Error]   An error occured whilst trying to perform a livepush: 
[Error]      ENOENT: no such file or directory, lstat '.goutputstream-4LFQ7Z'
```

The `.gooutputstream...` seems to be a temporary file, so when this is created, livepush will trigger (if a copy command will implicitly reference the directory containing this file, for example `COPY . .`) and if the file is removed before the copy occurs, livepush will fail.

**Comment by srlowe** [comment]
Ran into this issue myself with Ubuntu 19.10 - In my case it was happening when I modified files using `gedit`.  Other text editors did not show the same problem (both terminal (nano) and GUI based (Jetbrains Webstorm)).

**Comment by CameronDiver** [comment]
Yeah, I think the best way to handle this is to ignore when a file is added but is removed by the time livepush attempts to ship it into the running container. Honestly we could probably re-examine all of the failure modes when using livepush, I guess as a developement tool sometimes it can be a bit heavy handed in what it does not allow.
